packages/swarm-components/src/Button.jsx

```diff
@@ -4,112 +4,132 @@ import Icon from './Icon';
+	 */
 	icon?: string,
+
 	/**
-	* Inverted styles for dark backgrounds
-	*/
+	 * Inverted styles for dark backgrounds
+	 */
 	inverted?: boolean,
+
 	/**
-	* The neutral style
-	*/
+	 * The neutral style
+	 */
 	neutral?: boolean,
+
 	/**
-	* The function invoked when interacting with Button
-	*/
-	onClick?: (*) => void,
+	 * The function invoked when interacting with Button
+	 */
+	onClick?: () => {},
+
 	/**
-	* The primary style
-	*/
+	 * The primary style
+	 */
 	primary?: boolean,
+
 	/**
-	* Aligns the icon to the right
-	*/
+	 * Aligns the icon to the right
+	 */
 	right?: boolean,
+
 	/**
-	* Use the small button size
-	*/
+	 * Use the small button size
+	 */
 	small?: boolean,
+
+	/**
+	 * Content of button
+	 */
 	children: React.Node,
+
+	/**
+	 * Render an anchor styled as a button instead
+	 * of a button element
+	 */
+	isLink?: boolean,
+};
+
+// TODO: find a better, more dynamic solution
+const FILLS = {
+	default: 'var(--color-viridian)',
+	disabled: 'var(--color-gray-4)',
+	primary: 'var(--color-white)',
+	neutral: 'var(--color-gray-7)',
+	bordered: 'var(--color-gray-7)',
+	inverted: 'var(--color-white)',
 };
 
+
 /**
  *
  * @param {*} props
  *
  */
 export const getButtonType = (props: Props): string => {
 	let buttonType = 'default';
-	if (props.disabled) buttonType = 'disabled';
-	if (props.primary) buttonType = 'primary';
-	if (props.neutral) buttonType = 'neutral';
-	if (props.bordered) buttonType = 'bordered';
-	if (props.inverted) buttonType = 'inverted';
+
+	if (props.disabled) {
+		buttonType = 'disabled';
+	} else if (props.primary) {
+		buttonType = 'primary';
+	} else if (props.neutral) {
+		buttonType = 'neutral';
+	} else if (props.bordered) {
+		buttonType = 'bordered';
+	} else if (props.inverted) {
+		buttonType = 'inverted';
+	}
+
 	return buttonType;
 };
 
 export const getSwarmSize = (props: Props): string => {
-	let size = 'default';
-	if (props.small) size = 'small';
-	return size;
+	return props.small ? 'small' : 'default';
 };
 
 export const getIconPosition = (props: Props): string => {
-	let position = 'left';
-	if (props.right) position = 'right';
-	if (!props.children) position = 'only';
-	return position;
-};
-
-// TODO: find a better, more dynamic solution
-const FILLS = {
-	default: 'var(--color-viridian)',
-	disabled: 'var(--color-gray-4)',
-	primary: 'var(--color-white)',
-	neutral: 'var(--color-gray-7)',
-	bordered: 'var(--color-gray-7)',
-	inverted: 'var(--color-white)',
+	return (props.children) ? (props.right ? 'right' : 'left') : 'only';
 };
 
-const Button = (props: Props): React.Element<'button'> => {
+const Button = (props: Props): React.Element<'button' | 'a'> => {
 	// destructuring to not pass invalid attributes to node
 	const {
-		primary,
-		neutral,
-		bordered,
-		inverted,
-		small,
 		icon,
 		right,
 		children,
-		...rest
+		isLink,
+		...other
 	} = props;
 
 	if (props.disabled) {
 		delete props.onClick;
 	}
 
+	const Component = isLink ? 'a' : 'button';
```

Comment:
Can we just use `href` instead of `isLink`? It is a required prop for the case when we use `<a>` tag, right? I don't see a reason to introduce a separate `isLink` prop for it.